### PR TITLE
Remove filtersApplied from image-spec

### PR DESCRIPTION
**Pull request by imjasonh**
See https://github.com/opencontainers/distribution-spec/pull/380 which proposes changing how the registry communicates that it applied filters to be "out-of-band" of the image manifest response, so this doesn't need to be documented in image-spec at all.

Signed-off-by: Jason Hall <jason@chainguard.dev>

### CODE REVIEW

**Comment by sajayantony** [comment]
I feel this is a good change and as pointed out keeps the request semantics out of content/index. I feel we should merge this after  distribution spec changes have merged.

**Review by jonjohnsonjr — APPROVED** [review]

**Review by jonboulle — APPROVED** [review]
+1, I don't think this should have landed in the image spec. Let's keep it to distribution.

**Review by sajayantony — APPROVED** [review]

**Pull request merged**
